### PR TITLE
feat: Default event stories in storybook to action tab instead of controls tab

### LINKED ISSUE #59: Events stories in Storybook should default to the Actions tab
**The problem**
When looking at **Events** stories in Storybook, one would typically prefer to view the **Actions** tab of the story to see the events that emit when interacting with the component. Most Pharos stories default to the **Controls** tab, however.

**The solution**
Default all **Events** stories to the **Actions** tab.

### PR DESCRIPTION
**Pull request by satya-achanta-venkata**
**This change:** (check at least one)

- [x] Adds a new feature
- [ ] Fixes a bug
- [ ] Improves maintainability
- [x] Improves documentation
- [ ] Is a release activity

**Is this a breaking change?** (check one)

- [ ] Yes
- [x] No

**Is the:** (complete all)

- [x] Title of this pull request clear, concise, and indicative of the issue number it addresses, if any?
- [x] Test suite(s) passing?
- [x] Code coverage maximal?
- [x] Changeset added?

**What does this change address?**
Resolves #59 

**How does this change work?**
Default all Events stories to the Actions tab from controls tab.

### CODE REVIEW

**Review by daneah — COMMENTED** [review]
@satya-achanta-venkata Sorry if the desire wasn't explained well, but we don't want **all** the stories to default to the **Actions** tab. We only want the stories that have to do with **events** to default to that tab, because that's where the events will be visible. Other stories should continue defaulting to the **Controls** tab. I think this change overdoes it.

**Comment by satya-achanta-venkata** [comment]
@daneah  Got it. Thank you. Will do those changes.

**Review by Niznikr — COMMENTED** [review]
@satya-achanta-venkata Applying it to the `Meta` element applies it to all the stories. We only want to set `addon-actions` for the individual stories that deal with events and use the `Story` element.

**Comment by daneah** [comment]
@satya-achanta-venkata @Niznikr I'm thinking it'd make sense to ensure all stories default to the **Controls** tab (using `Meta`) and then override this specifically for the event stories. Thoughts?

**Comment by satya-achanta-venkata** [comment]
@daneah @Niznikr makes sense. I understand what you mean. Will make those changes.

**Review by daneah — APPROVED** [review]
This looks right to me—thanks @satya-achanta-venkata for the iteration on this!

**Review by Niznikr — APPROVED** [review]

File: .changeset/angry-mugs-double.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@ithaka/pharos': minor
+---
+
+Update stories in storybook to default to actions tab instead of controls tab
```
Comment (Niznikr) [inline .changeset/angry-mugs-double.md:5]:
```suggestion
update event stories in Storybook to default to actions tab instead of controls tab
```

**Pull request merged**